package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "heroku-pg",
   "description": "Heroku CLI plugin to manage Postgres.",
-  "version": "2.4.2",
+  "version": "2.5.0",
```

Comment:
The version should be bumped separately and that's done automatically by `np`: see https://github.com/heroku/heroku-pg#publishing